examples/jsm/cameras/CinematicCamera.js

```diff
@@ -13,205 +13,205 @@ import {
 import { BokehShader } from "../shaders/BokehShader2.js";
 import { BokehDepthShader } from "../shaders/BokehShader2.js";
 
-var CinematicCamera = function ( fov, aspect, near, far ) {
+class CinematicCamera extends PerspectiveCamera {
```

Comment:
This PR is not mergeable as long as you have to modify example code.

Notice that it is not acceptable to just modify `jsm` modules. Until end of this year, `example/js` and `exampes/jsm` have to stay in sync (expect for new files). 

Please revert any changes to `CinematicCamera` and `PerspectiveCamera`.

Reply:
How about this https://github.com/mrdoob/three.js/pull/20102

Reply:
Closing in favor of #20102.